superbench/cli/_handler.py

```diff
@@ -72,51 +127,90 @@ def deploy_command_handler(
         raise CLIError('Must specify both docker_username and docker_password if authentication is needed.')
     if not (host_file or host_list):
         raise CLIError('Must specify one of host_file or host_list.')
-    check_argument_file('host_file', host_file)
-    check_argument_file('private_key', private_key)
+    host_file = check_argument_file('host_file', host_file)
+    private_key = check_argument_file('private_key', private_key)
 
-    raise NotImplementedError
+    # Docker config
+    docker_config = OmegaConf.create()
+    for key in ['image', 'username', 'password']:
+        docker_config[key] = eval('docker_{}'.format(key))
+    docker_config['registry'], _ = split_docker_domain(docker_image)
+    # Ansible config
+    ansible_config = OmegaConf.create()
+    for key in ['file', 'list', 'username', 'password']:
```

Comment:
Why not write 'host_file', 'host_list', 'host_username ', and 'host_password' directly? It would be easy to read and replace the variable